ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/match-list-input.md

```diff
@@ -18,6 +26,22 @@
 
     ```yaml
     match_list_input:
+      prefix_ipv4:
+
+          # Prefix-List Name
+        - name: <str; required; unique>
+
+          # List of IPv4 Prefixes (with the subnet mask e.g. 10.10.10.0/24)
+          prefixes: # required
+            - <str>
+      prefix_ipv6:
+
+          # Prefix-List Name
+        - name: <str; required; unique>
+
+          # List of IPv6 Prefixes (with the subnet mask e.g. 2001:db8:abcd:0013::/64)
+          prefixes: # required
+            - <str>
```

Comment:
These lists of strings often come back to haunt us :) Please make sure the CLI does not offer any options that would be natural to put here. Basically if we should make this a a list of dicts.

Reply:
I can't see anything - but time will tell I guess. I've left it as a list of strings for now...